server/modules/tasks.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  addSyncMap,
+  addSyncMapFilter,
+  BaseServer,
+  ChangedAt,
+  NoConflictResolution
+} from '@logux/server'
+import { defineSyncMapActions, LoguxNotFoundError } from '@logux/actions'
+
+import {
+  changeTask,
+  createTask,
+  deleteTask,
+  findTask,
+  getUserTasks
+} from '../db.js'
+import { Task } from '../../api/index.js'
+
+const modelName = 'tasks'
+
+const [createTaskActionType] = defineSyncMapActions(modelName)
+
+export default (server: BaseServer): void => {
+  addSyncMap<Task>(server, modelName, {
+    async access(ctx, id, action) {
+      if (createTaskActionType.type === action.type && 'fields' in action) {
+        return ctx.userId === action.fields.authorId
+      } else {
+        const task = await findTask(id)
+        return ctx.userId === task?.authorId
```

Comment:
```suggestion
        return task && ctx.userId === task.authorId
```

Let’s make test more clear. It is always better for security.